tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -107,11 +107,10 @@ func newSession(credentialPath, credentialAccountID, region string) (*awssession
 	if _, err := os.Stat(credentialPath); os.IsNotExist(err) {
 		return nil, fmt.Errorf("failed to find AWS credentials from path '%v'", credentialPath)
 	}
-	session := awssession.Must(awssession.NewSession(&aws.Config{
+	return awssession.NewSession(&aws.Config{
```

Comment:
Please mention in the commit message how moving from `Must` to `NewSession` fixes the issue.